### DISCUSSION TITLE
WIN10系统下载了RIME后，不能打中文，只能用英文，这该怎么办？

### DISCUSSION THREAD

**Discussion by azhongzz** [rime/home#1342] [Q&A]
WIN10系统下载了RIME后，不能打中文，打字只会弹出输入框，但是没有相应的中文输出

**Comment by CN-make** [comment]
我也是 今天 出现这个 问题不知道是不是作者坚持不下去关闭了服务器

**Reply by liuliping0315** [reply]
输入法是本地程序，不需要联网功能。

**Reply by Gaulhuiun** [reply]
wo de ye shi

**Reply by liuliping0315** [reply]
请尝试从开始菜单所有应用里找到“小狼毫输入法”文件夹，点开，并点击 小狼毫算法服务。

**Reply by gshmu** [reply]
我一般是菜单里 deploy 点击后就好了
